.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "haskell.haskell"
```

Comment:
Unfortunately this doesn't really work. And it definitely won't work (for a while) if I upgrade to GHC 9.2.